From adebb027be5d091cf3368ec27689e7bdc288095e Mon Sep 17 00:00:00 2001
From: Vincent Blut <vincent.debian@free.fr>
Date: Tue, 1 Sep 2020 14:01:10 +0200
Subject: sys_linux: allow readlinkat in seccomp filter


diff --git a/sys_linux.c b/sys_linux.c
index 2555e8b..af45066 100644
--- a/sys_linux.c
+++ b/sys_linux.c
@@ -537,6 +537,7 @@ SYS_Linux_EnableSystemCallFilter(int level, SYS_SystemCallContext context)
     SCMP_SYS(lstat64),
     SCMP_SYS(newfstatat),
     SCMP_SYS(readlink),
+    SCMP_SYS(readlinkat),
     SCMP_SYS(rename),
     SCMP_SYS(renameat),
     SCMP_SYS(renameat2),
-- 
cgit v0.10.2

