From 1f8355f154d6204628da4f4dfea6064082ae14c1 Mon Sep 17 00:00:00 2001
From: Miroslav Lichvar <mlichvar@redhat.com>
Date: Tue, 17 Mar 2020 15:56:31 +0100
Subject: test: make 139-nts more reliable


diff --git a/test/simulation/139-nts b/test/simulation/139-nts
index aa5b0bd..e538fb8 100755
--- a/test/simulation/139-nts
+++ b/test/simulation/139-nts
@@ -43,7 +43,7 @@ check_chronyd_exit || test_fail
 check_source_selection || test_fail
 check_sync || test_fail
 
-check_file_messages "20.*123\.1.* 111 111 1111" 91 93 measurements.log || test_fail
+check_file_messages "20.*123\.1.* 111 111 1111" 90 93 measurements.log || test_fail
 check_file_messages "20.*123\.1.* 111 001 0000" 30 32 measurements.log || test_fail
 rm -f tmp/measurements.log
 
@@ -55,11 +55,12 @@ check_chronyd_exit || test_fail
 check_source_selection || test_fail
 check_sync || test_fail
 
-check_file_messages "20.*123\.1.* 111 111 1111" 101 103 measurements.log || test_fail
+check_file_messages "20.*123\.1.* 111 111 1111" 99 103 measurements.log || test_fail
 check_file_messages "20.*123\.1.* 111 001 0000" 0 0 measurements.log || test_fail
 rm -f tmp/measurements.log
 
-client_conf=""
+client_conf="
+nosystemcert"
 
 run_test || test_fail
 check_chronyd_exit || test_fail
-- 
cgit v0.10.2

